bookkeeper-dist/src/main/resources/LICENSE-server.bin.txt

```diff
@@ -444,12 +445,6 @@ Bundled as
   - lib/com.google.auth-google-auth-library-credentials-0.9.0.jar
 Source available at https://github.com/google/google-auth-library-java/tree/0.9.0
 ------------------------------------------------------------------------------------
-This product bundles the JLine Library, which is available under a "2-clause BSD"
-license. For details, see deps/jline-2.11/LICENSE
-
-Bundled as
-  - lib/jline-jline-2.11.jar
```

Comment:
is it possible the the problem is related to the lack of "jline" on the classpath ?